nvflare/private/fed/client/fed_client_base.py

```diff
@@ -209,6 +209,9 @@ def _create_cell(self, location, scheme):
         self.cell.start()
         self.communicator.cell = self.cell
         self.net_agent = NetAgent(self.cell)
+        mpm.add_cleanup_cb(self.net_agent.close)
```

Comment:
The reason for the not-stop is because the MPM hang without the cleanup_cb after the cell started when no client_runner created. Move it earlier to ensure register the cleanup_cb right after the cell start.